### PR TITLE
[TASK] Hint at symlink creation in Copy assets chapter

### PR DESCRIPTION
**Pull request by linawolf**
Resolves: https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-SitePackage/issues/207
Releases: main, 13.4

### CODE REVIEW

File: Documentation/Assets/Index.rst [hunk 1]
```diff
@@ -54,6 +54,23 @@ we are finished with the site package.
...
+
+Symlinking Resources/Public into public/_assets
+===============================================
+
+When creating your :ref:`Minimal site package <t3sitepackage:minimal-design>`
+your extension did not yet have a folder :path:`Resources/Public`. The symlink
+from :folder:`public/_assets` gets automatically created during Composer
+installation.
+
+However as you newly created the folder, you must tell Composer to dump the
```
Comment (garvinhicking) [inline Documentation/Assets/Index.rst:67]:
```suggestion
However as you newly created the folder, you must tell Composer to re-perform this initialization-process, which is done in the "dump-autoload" step.
```

File: Documentation/Assets/Index.rst [hunk 2]
```diff
@@ -54,6 +54,23 @@ we are finished with the site package.
...
+autoload. During that process the symlinks will also be created by Composer.
```
Comment (garvinhicking) [inline Documentation/Assets/Index.rst:68]:
```suggestion
During that process the symlinks will also be created by Composer.
```

**Review by garvinhicking — COMMENTED** [review]
I tried a small rephrase, what do you think?

**Comment by linawolf** [comment]
> I tried a small rephrase, what do you think?

Sounds good to me, thank you!

**Review by garvinhicking — APPROVED** [review]

**Pull request merged**